config/settings/production.py

```diff
@@ -287,3 +268,9 @@
 
 # django-defender configuration
 DEFENDER_REDIS_NAME = "default"
+
+if REDIS_LOCATION.startswith("rediss://"):
```

Comment:
Should this just have one 's'?

```suggestion
if REDIS_LOCATION.startswith("redis://"):
```

Reply:
No, it's right. From https://devcenter.heroku.com/articles/heroku-redis:

> As of Redis version 6, Hobby plans support both TLS and unencrypted connections, while production plans require TLS connections. For hobby plans you can find the unencrypted (plaintext) and TLS URIs in your app’s config vars REDIS_URL and REDIS_TLS_URL respectively. For Premium, Private, and Shield plans on version 6 your REDIS_URL config var will use the rediss: scheme, and you must enable TLS in your Redis client’s configuration in order to connect to a Redis 6 database.